### PR TITLE
Add Zooz ZSE42 v1.5

### PR DESCRIPTION
**Pull request by MYanello**

### CODE REVIEW

**Comment by AlCalzone** [comment]
@TSHAdmin please review this.

File: firmwares/zooz/ZSE42.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+			"productId": "0xe002",
+			"firmwareVersion": {
+				"min": "1.0",
+				"max": "1.255"
+			}
+		}
+	],
+	"upgrades": [
+		{
+			"version": "1.50",
```
Comment (TSHAdmin) [inline firmwares/zooz/ZSE42.json:17]: version format: x.y.z
for Zooz product details contact @[TSHAdmin]

Comment (MYanello) [reply firmwares/zooz/ZSE42.json:17]: Happy to change the format, but here we have version 1.50 https://www.support.getzooz.com/kb/article/1158-zooz-ota-firmware-files/

Is the zooz firmware page using the wrong firmware format?

Comment (TSHAdmin) [reply firmwares/zooz/ZSE42.json:17]: For [ZSE42_V01R50_US.gb](https://www.getzooz.com/firmware/ZSE42_V01R50_US.gbl)l firmware version is 1.50.1
If you like to get more details on versioning of Zooz products contact @[TSHAdmin]

Comment (myanello-rescale) [reply firmwares/zooz/ZSE42.json]: Thank you, added the patch version.

**Review by TSHAdmin — APPROVED** [review]
Thank you or the contribution.
File name and content approved.

**Comment by MYanello** [comment]
@AlCalzone can you kick off the workflows for this please?

**Comment by AlCalzone** [comment]
Closing and reopening to trigger workflows

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**